p8_integration_tests/test_multi_call_examples/test_synfire_1_run_reset_1_run__larger_runtime_no_extraction_if_curr_exp.py

```diff
@@ -7,6 +7,7 @@
 import spynnaker.plot_utils as plot_utils
 import spynnaker.spike_checker as spike_checker
 from spynnaker8.utilities import neo_convertor, neo_compare
+from spynnaker8.utilities.version_util import pynn8_syntax as pynn8_syntax
```

Comment:
Why the `import blah as blah`? That's just wordy

Reply:
fixed